pkg/build/node/node.go

```diff
@@ -255,7 +255,7 @@ func (c *BuildContext) buildImage(dir string) error {
 		}()
 	}
 	if err != nil {
-		log.Errorf("Image build Failed! %v", err)
+		log.Errorf("Image build Failed! Failed to create build container: %v", err)
```

Comment:
I wound up adding some more details to these to aid debugging while working on this. They're not strictly related, but it's a small change.